lib/messages.js

```diff
@@ -3,7 +3,8 @@ import cstruct from 'compact-encoding-struct'
 
 export const AuthZResponse = cstruct.compile({
   status: c.string,
-  message: cstruct.opt(c.string)
+  message: cstruct.opt(c.string),
+  feed: cstruct.opt(c.string)
```

Comment:
Since this is AuthZ (I assume stands for authorization) it is perfectly reasonable expectations to have list of "things" to which user has authorized accessed in case of successful authorization. Therefore, I suggest changing it to:
```
export const AuthZResponse = cstruct.compile({
  status: c.string,
  message: cstruct.opt(c.string),
  resources: cstruct.array(c.string) // or: cstruct.opt(cstruct.array(c.string))
})
```

Reply:
I understand adding such extra field, but the purpose is that Bitkit can associate a feed with the account in the same widget, so there has to be a deterministic place for Bitkit to look for, otherwise we will need ad-hoc logic like filtering resources for urls that look like a feed, and take the first one? the last one?

reference task: https://app.asana.com/0/1200529269167147/1204330251427305

Reply:
url's contain `protocol`, so I expect bitkit to be able to use it for this purpose

Reply:
yes, since widget's does not support multiple feeds (yet?) taking the fist one seem reasonable